doc/whats_new/v0.22.rst

```diff
@@ -15,6 +15,17 @@ This is a bug-fix release to primarily resolve some packaging issues in version
 Changelog
 ---------
 
+:mod:`sklearn.inspection`
+.........................
+
+- |Fix| :func:`inspection.permutation_importance` will return the same
+  `importances` when a `random_state` is given for both `n_jobs=1` or
+  `n_jobs>1` both with shared memory backends (thread-safety) and
+  isolated memory, process-based backends.
+  Also avoid casting the data as object dtype and avoid read-only error
+  on large dataframes with `n_jobs>1` as reported in :issue:`15810`.
+  :pr:`15933` by :user:`Guillaume Lemaitre <glemaitre>` and `Olivier Grisel`_.
```

Comment:
Maybe we could acknowledge @shivamgargsya  's work for https://github.com/scikit-learn/scikit-learn/pull/15898 here (even if it's a more general solution)

Reply:
Thanks. This is a good point. I'll make the change